components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConsumerImpl.java

```diff
@@ -3700,6 +3972,60 @@ public OAuthApplicationInfo updateAuthClient(String userId, String applicationNa
 
     }
 
+    /**
+     * @param userId Subscriber name.
+     * @param applicationName of the Application.
+     * @param applicationId of the Application.
+     * @param tokenType Token type (PRODUCTION | SANDBOX)
+     * @param callbackUrl callback URL
+     * @param allowedDomains allowedDomains for token.
+     * @param validityTime validity time period.
+     * @param groupingId APIM application id.
+     * @param jsonString Callback URL for the Application.
+     * @param tokenScope Scopes for the requested tokens.
+     * @return
+     * @throws APIManagementException
+     */
+    @Override
+    public OAuthApplicationInfo updateAuthClientByAppId(String userId, String applicationName, int applicationId,
+            String tokenType,
```

Comment:
do we need separate lines for each variable?

Reply:
Fixed with 89b6956b5c1641bd96ee670c827bbf6d0a87550a